Assets/Thirdweb/Examples/Scripts/Prefabs/Prefab_ConnectWallet.cs

```diff
@@ -31,8 +32,8 @@ public struct NetworkSprite
 public class Prefab_ConnectWallet : MonoBehaviour
 {
     [Header("SETTINGS")]
-    public List<Wallet> supportedWallets = new List<Wallet> { Wallet.MetaMask, Wallet.CoinbaseWallet, Wallet.WalletConnect };
-    public bool supportSwitchingNetwork = false;
+    public List<Wallet> supportedWallets;
```

Comment:
we still have the same defaults right?

Reply:
defaults are set in the prefab from inspector so yes